packages/storybook/src/stories/ino-range/ino-range.stories.ts

```diff
@@ -39,24 +51,15 @@ export const Playground: Story<Components.InoRange> = (args) => html`
     markers="${args.markers}"
     value="${args.value}"
     step="${args.step}"
-    color-scheme="${args.colorScheme}"
-  >
-  </ino-range>
-`;
+    color-scheme="${args.colorScheme}"></ino-range>
+`);
+export const Playground = template.generatePlaygroundStory();
+export const DiscretRange = template.generateStoryForProp('discrete', true);
+export const SteppedRange = template.generateStoryForProp('step', 25);
```

Comment:
You can move lines 113 to 117 to this line:
```suggestion
export const SteppedRange = template.generateStoryForProp('step', 25, {
  discrete: true,
  markers: true,
});
```